ulid/test.ts

```diff
@@ -0,0 +1,233 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// Copyright 2023 Yoshiya Hinosawa. All rights reserved. MIT license.
+// Copyright 2017 Alizain Feerasta. All rights reserved. MIT license.
+
+import { FakeTime } from "../testing/time.ts";
+import {
+  assert,
+  assertEquals,
+  assertStrictEquals,
+  assertThrows,
+} from "../assert/mod.ts";
+
+import * as ULID from "./mod.ts";
+const ulid = ULID.factory();
+
+Deno.test("ulid", async (t) => {
```

Comment:
Nit: I don't see the benefit of having this extra nesting level with tests. It could be more flat if done as separate `Deno.test()`s. Not a strong opinion, though.